web/src/features/charts/elements/AreaGraph.tsx

```diff
@@ -105,12 +105,12 @@ function AreaGraph({
   tooltip,
   tooltipSize,
 }: AreagraphProps) {
-  const {
-    ref,
-    width: containerWidth,
-    height: containerHeight,
-    node,
-  } = useReferenceWidthHeightObserver(Y_AXIS_WIDTH, X_AXIS_HEIGHT);
+  const reference = useRef(null);
+  const { width: observerWidth = 0, height: observerHeight = 0 } =
```

Comment:
and here?

These will become negative initially and well negative width and height don't really work.